### DISCUSSION TITLE
I am unable to fetch Ads Marketing stats!

### DISCUSSION THREAD

**Discussion by developereleonor** [etsy/open-api#1315] [Q&A]
Hello Support,

I am stuck with an issue to on my integration via Etsy's API.

So basically, I checked the API and want to retrieve Ads Marketing stats (including Ads Cost) from Etsy Store using the API. But I didn't get any end-points in the API to get the same. So please let me know if there are any advanced APIs, or API accesses or any other way to get the same in my website from the Etsy store.

I see that Etsy gives it's Ads Marketing stats/data to ProfitTree, so there should be some way to get that data from any end-points too.

This is really important for us, so a prompt response will be helpful.

Thanks!

**Comment by turboturtleD** [comment]
This is more of a Etsy support group than actual Etsy support.  You would need to contact developer@etsy.com.

As far as I know, however, there is no accessible endpoint for the data you seek.  Etsy has partnerships with some larger sites where clearly agreements were struck to have access to otherwise unavailable data.  I think these exceptions are few and far between.

**Reply by BCham457** [reply]
Would you mind telling me some of these sites? In particular, if there is a site/service which allows me to get ad spend/ad revenue for each day over a given date range. I would be really interested in that.

**Comment by cyrilchandelier** [comment]
You can get the global ad spend per day by going through the ledger entries and summing the amounts associated with `prolist` and `offsite_ads_fee` entries. There is no endpoint in the official API to get this at the listing level though.

The company you mentioned is retrieving ads data via a sort of scraping happening outside of the API, which is really not recommended if you want to stay in good standing with Etsy. And this type of practices is getting them in trouble I think, I heard they recently got their API access temporarily suspended, I'm guessing that's related.
